opam

```diff
@@ -27,10 +27,11 @@ depends: [
   "menhir"       {>= "20160303"}
   "merlin"       {>= "2.5.0"}
   "merlin-extend"{>= "0.3"}
+  "result"
```

Comment:
It is correct to depend on result on 4.02 (build is passing on 4.02.3 right now)